source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,310 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page you will learn the how to search **geospatial data** in the
+MongoDB Java Driver.
+
+Geospatial data is data that represents a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
+-----------------------------------
+
+You can represent geospatial data on an Earth-like sphere with **GeoJSON**.
+GeoJSON is a geospatial data format created by the Internet Engineering
+Task Force (IETF). You can use the GeoJSON format to store your data in MongoDB,
+and you must use the GeoJSON format to query your geospatial data in MongoDB. 
+
+For a concrete reference, GeoJSON data often resembles the following structure:
```

Comment:
Is this how it's read/stored in MongoDB? If so, would it be helpful to focus on that information rather than describing it as "a concrete reference... often resembles"?

Reply:
Thanks for the note. In line with our discussion in team PR review, this was changed into the location of MongoDB headquarters. https://github.com/mongodb/docs-java/pull/78/commits/5f31b46a5700bb7bf4a78504a2b09b2b3253b629

I agree that the "concrete reference... often resembles" is not productive (and sort of nonsensical). I agree with your and Kailie's comments that just presenting the info is better here (and providing an actual concrete reference of an understandable place is better).